src/auth/models.py

```diff
@@ -1,35 +1,42 @@
-from sqlalchemy import Column, DateTime, Integer, String
-# from sqlalchemy.orm import relationship
-
+from constants import COMPUTING_ID_LEN, SESSION_ID_LEN
 from database import Base
+from sqlalchemy import Column, DateTime, ForeignKey, String
+from sqlalchemy.orm import relationship
 
 
 class UserSession(Base):
     __tablename__ = "user_session"
 
     # note: a primary key is required for every database table
-    id = Column(Integer, primary_key=True)
-
-    issue_time = Column(DateTime, nullable=False)
-    session_id = Column(String(512), nullable=False)  # the space needed to store 256 bytes in base64
     computing_id = Column(
-        String(32), nullable=False
+        String(COMPUTING_ID_LEN), nullable=False, primary_key=True, unique=True
```

Comment:
as said above unique=True is redundant

Reply:
fixed! https://github.com/CSSS/csss-site-backend/pull/26/commits/5b3a3a8e41149355d1b16c6c3375049759199b1c